app/src/assets/css/style.scss

```diff
@@ -16,4 +16,6 @@
 @import './modules/_section.scss';
 @import './modules/_utility.scss';
 @import './modules/_vega.scss';
-@import './modules/_visualize.scss';
\ No newline at end of file
+@import './modules/_visualize.scss';
+@import './modules/editimage.scss';
```

Comment:
I think these should be _editimage and _imageupload with underscore?

Reply:
Huh, you're right but they worked anyway. Weird. I fixed it in 3c4acba.